src/pages/MyStorage/components/Files/FileAdapter.tsx

```diff
@@ -1,3 +1,4 @@
+import { Dialog } from "@headlessui/react";
```

Comment:
hey @borjasolerme 
don't need to install this frontend componens npm

we have default modal component!

![image](https://github.com/Hello-Storage/hello-front/assets/97012368/fc138d91-634d-4e18-9979-7ed9acea1abd)

check above image